### PR TITLE
Issue #66 - Add icon active state

### PR DESCRIPTION
**Pull request by FrancescoSTL**
Adds active state for icon actionButton

issue ref: #66 

r? @schalkneethling

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -519,6 +524,12 @@ function toggleSidebar() {
     allAboard.state('window', {
         badge: null
```
Comment (schalkneethling) [inline index.js]: You can do the below here i.e.
```
badge: null,
icon: {
    ...
}
```
It is also the documented way to do this https://developer.mozilla.org/en-US/Add-ons/SDK/Low-Level_APIs/ui_button_action#state()

File: index.js [hunk 2]
```diff
@@ -162,6 +162,11 @@ function showBadge() {
         badge: '1',
         badgeColor: '#5F9B0A'
     });
+    allAboard.icon = {
+        '16': './media/icons/icon-16_active.png',
+        '32': './media/icons/icon-32_active.png',
+        '64': './media/icons/icon-64_active.png'
+    };
```
Comment (schalkneethling) [inline index.js]: As mentioned below, move this up into the `state` update section.

**Comment by FrancescoSTL** [comment]
@schalkneethling fixed nits. r?

**Comment by schalkneethling** [comment]
r+ 📛

**Pull request merged**